app/views/accounts/_form.html.erb

```diff
@@ -4,11 +4,9 @@
     <%= form.label "Name", style: "display: block" %>
     <%= form.text_field :name, required: true, class: "form-control"%>
   </div>
-
   <div class="form-group">
     <%= form.label "Amount", style: "display: block" %>
     <%= form.number_field :amount, required: true, class: "form-control" %>
   </div>
-
   <%= form.submit "Create Transaction", class: "btn btn-primary mt-4"%>
```

Comment:
- [ ] In the `Add a new transaction` page, we are required to fill out 3 mandatory inputs as we can see in the project requirements below. However, we are missing the `categories` input field
![Screenshot from 2023-04-24 22-10-43](https://user-images.githubusercontent.com/46388113/234127101-8fc124f2-defb-44f2-bc3b-094c3470cff1.png)